### DISCUSSION TITLE
Better keybind error messages

### DISCUSSION THREAD

**Discussion by otherJL0** [ghostty-org/ghostty#3732] [Feature Requests, Ideas] [answered]
When editing the config file, any errors in keybind are vague and don't provide enough context to quickly determine where the issue is located:

![image](https://github.com/user-attachments/assets/babfaad3-371f-4381-9559-65f84e84c607)

I think the error messages for keybind errors should be more descriptive, something like:

![image](https://github.com/user-attachments/assets/4c5f2151-9f67-436a-a4af-21a57c528f7c)

I have included a [draft pr](https://github.com/ghostty-org/ghostty/pull/3727) for consideration

**Comment by rhodes-b** [comment, marked as answer]
https://github.com/ghostty-org/ghostty/issues/5333 marking this the tracking issue for this
